dev/deploy.rst

```diff
@@ -22,6 +22,31 @@ languages!
 
 .. _manual_code_deploy:
 
+Deploy Artifacts
+----------------
+
+During build, the deploy process will attempt to generate a ``deploy.json`` that can provide
```

Comment:
nit: it's not really a 'build' ... it's a deploy

Reply:
Yeah, I put build in qoutes too on the other repo, I'll change the wording.